styles/modules/_product.scss

```diff
@@ -0,0 +1,60 @@
+.product {
```

Comment:
moved to alpaca-components in https://github.com/SnowdogApps/magento2-alpaca-ui-components/pull/43